src/app/cli/src/init/client.ml

```diff
@@ -1029,6 +1029,38 @@ let to_signed_fee_exn sign magnitude =
   let sgn = match sign with `PLUS -> Sgn.Pos | `MINUS -> Neg in
   Currency.Fee.Signed.create ~sgn ~magnitude
 
+let format_pending_snark_work ?(f = Fun.id) graphql_endpoint =
```

Comment:
@mrmr1993 has sent me nits,about passing in functions like this instead of piping, but I don't really mind.